docker-cloud.yml

```diff
@@ -89,4 +89,19 @@ specstore:
   expose:
     - '8000'
   image: 'datopian/specstore:latest'
-
+elasticsearch:
+  autoredeploy: true
+  image: 'elasticsearch:latest'
```

Comment:
Is this the elasticsearch which has been deprecated? https://hub.docker.com/_/elasticsearch/